primitives/runtime/src/testing.rs

```diff
@@ -260,13 +260,13 @@ pub struct TestXt<Call, Extra> {
 	/// Signature of the extrinsic.
 	pub signature: Option<(u64, Extra)>,
 	/// Call of the extrinsic.
-	pub call: Call,
+	pub function: Call,
```

Comment:
impl_outer_inherent make use of this field directly :-/, I guess best is to add a method to Extrinsic trait. I'm doing it now

Reply:
done